SwiftRustIntegrationTestRunner/test-release-fails/main.swift

```diff
@@ -0,0 +1,4 @@
+import Foundation
+import TestReleaseFails
+
+call_swift_add()
```

Comment:
Maybe `call_rust()` to make it clear that we're calling into Rust.

Right now it seems like we're calling a Swift function.